lib/auth/accountrecovery.go

```diff
@@ -117,13 +103,6 @@ func (a *Server) verifyCodeWithRecoveryLock(ctx context.Context, username string
 		return trace.AccessDenied(startRecoveryGenericErrMsg)
 	}
 
-	status := user.GetStatus()
-	if status.IsLocked && status.RecoveryAttemptLockExpires.After(a.clock.Now().UTC()) {
```

Comment:
Can we delete the Login.RecoveryAttemptLockExpires field?

Reply:
We can, however I was trying to avoid protocol changes until the next major version.  Can I simply remove this field now?

Reply:
It depends on how the field is usage. We can deprecate it and remove in the following major, but sometimes outright removal is possible.